src/components/VideoPlayer/VideoPlayer.jsx

```diff
@@ -0,0 +1,26 @@
+import { faAngleLeft } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { BackToVideosButton, IFameContainer, VideoContainer, VideoDetailsContainer } from './VideoPlayer.styled';
+
+
+
+const VideoPlayer = ({ videoData, setVideoId }) => {
+    const videoItem = videoData?.items[0];
+
+    return (
+        <VideoDetailsContainer>
+            <VideoContainer>
+                <BackToVideosButton onClick={() => setVideoId(null)}>
+                    <FontAwesomeIcon icon={faAngleLeft} size="lg" />
+                    &nbsp;Back to videos
+                </BackToVideosButton>
+                <IFameContainer dangerouslySetInnerHTML={{ __html: videoItem.player.embedHtml }} />
```

Comment:
En este caso no es del todo `dangerous`, pero igual te recomiendo leer un poco de los posibles riesgos que podría tener
https://stackoverflow.com/questions/29044518/safe-alternative-to-dangerouslysetinnerhtml

Reply:
Entiendo, voy a informarme al respecto 👍